bosh-stemcell/spec/stemcells/fips_spec.rb

```diff
@@ -2,7 +2,11 @@
 
 describe 'FIPS Stemcell', os_image: true do
   context 'installed by system_kernel' do
-    describe package('linux-image-fips') do
+    iaas = ENV['IAAS_KERNEL'] || ''
+    unless iaas.empty?
```

Comment:
is this check for empty required? isn't a check for `ENV['IAAS_KERNEL']` enough? at least that's what's done in other places in this PR.

Reply:
I need to append that hyphen if `IAAS_KERNEL` isn't empty otherwise I want an empty string.

I could drop this in favor of the following oneliner
```rb
iaas = ("#{ENV['IAAS_KERNEL']}-" if ENV['IAAS_KERNEL']) || ''
```

ruby is also not my forte, so maybe I'm missing a more obvious way to do this.

Reply:
thx (looks like I can't "resolve" this thread.